maestro/audit.py

```diff
@@ -361,22 +362,40 @@ def from_config(cfg):
 class ExecuteScriptAuditor(BaseAuditor):
     """Auditor that executes scripts with maestro action information."""
 
-    def __init__(self, name, script, level):
+    def __init__(self, args, script, level):
```

Comment:
Nit: swap `args` and `script`. It's more natural to be thinking the script first, then the args to the script.